app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,32 @@
+class AnswersController < ApplicationController
+
+  def new
+    @answer = Answer.new
+  end
+
+  def create
+    @question = Question.find_by(id: params[:question_id])
+    @answer = @question.answers.new(answer_params)
+    @answer.update_attributes(user: current_user)
+    if @answer.save
+      # respond_to do |format|
+      #   format.js do
+          redirect_to question_path(params[:question_id])
+          # render partial: 'add_ajax_answer' and return
+        # end
+        # format.any do
+        #   redirect_to question_path(params[:question_id])
+      #   end
+      # end
```

Comment:
Protip: When doing something like this (removing all the code of a feature you're temporarily disabling) it's better do do the work in a single commit and just delete everything. That way you have one commit to see all of the changes, you haven't polluted your repository with commented out code that might never be needed and if you DO want to bring it back, you can revert the commit more easily than finding and uncommenting all of this code.